### PR TITLE
fix(admin): notifications sending enabled control changed to toggle

### PR DESCRIPTION
**Pull request by kofzera**

### CODE REVIEW

**Review by xflord — CHANGES_REQUESTED** [review]
Looks good, but part of the task was also to add a success notification when the 'sending setting' changes (similarly as a notification pops up when you add/remove entities). You can do this using the `NotificatorService`

**Comment by kofzera** [comment]
> Looks good, but part of the task was also to add a success notification when the 'sending setting' changes (similarly as a notification pops up when you add/remove entities). You can do this using the `NotificatorService`

Somehow forgot about that, should be there now :smile:

File: apps/admin-gui/src/app/vos/components/notification-list/notification-list.component.ts [hunk 1]
```diff
@@ -99,12 +108,14 @@ export class NotificationListComponent implements OnChanges, AfterViewInit {
         .setSendingEnabled({ mails: [applicationMail], enabled: false })
         .subscribe(() => {
           applicationMail.send = false;
+          this.notificator.showSuccess(this.disabledSendingMessage);
```
Comment (bodnara) [inline apps/admin-gui/src/app/vos/components/notification-list/notification-list.component.ts:111]: Preferably use the `showInstantSuccess` method of the notificator. It handles the translation itself.

**Comment by sarkapalkovicova** [comment]
In `add-edit-notification-dialog` is also used a checkbox. Shouldn't be here toggle too? @bodnara

**Comment by bodnara** [comment]
> In `add-edit-notification-dialog` is also used a checkbox. Shouldn't be here toggle too? @bodnara

In this case checkbox is better option, since the action requires conformation (click on save button). In case of immediate action toggle is preferable.

**Review by sarkapalkovicova — APPROVED** [review]

**Pull request merged**